crates/ruff/src/rules/flake8_builtins/rules/builtin_attribute_shadowing.rs

```diff
@@ -81,12 +83,39 @@ pub(crate) fn builtin_attribute_shadowing(
             return;
         }
 
+        checker.diagnostics.push(Diagnostic::new(
+            BuiltinAttributeShadowing {
+                name: name.to_string(),
+            },
+            range,
+        ));
+    }
+}
+
+/// A003
+pub(crate) fn builtin_method_shadowing(
```

Comment:
(Same violation, but split into one method for attributes, another for methods.)